src/editor/components/components/ActionBar/ActionBar.component.jsx

```diff
@@ -28,6 +29,19 @@ const ActionBar = ({ handleAddClick, isAddLayerPanelOpen }) => {
     setCursorEnabled(true);
   };
 
+  const handleCameraCLick = () => {
+    Events.emit('camera_clicked');
+    const screenshotEl = document.getElementById('screenshot');
+    screenshotEl.play();
```

Comment:
Is this really needed? I don't see any play function or events in the screentock component.

Reply:
i just copy pasted what was in the other function!